plugins/sinks/http/README.md

```diff
@@ -10,19 +10,58 @@ sinks:
   config:
     method: POST
     success_code: 200
-    url: https://compass.com/v1beta1/asset
+    url: https://compass.requestcatcher.com/v1beta2/asset/{{ .Type }}/{{ .Urn }}
```

Comment:
what is this requestcatcher, and what is the differences between v1beta1 and v1beta2 APIs?

Reply:
just a dummy example, don't really mind about it